### ISSUE TITLE
Is there anyway to run `serverless offline` without generate ".build" directory?

### ISSUE THREAD

**Issue by tomzaku** [prisma/serverless-plugin-typescript#166]

**Comment by KingDarBoja** [comment]
I don't think so, as the package does need to bundle the lambda .ts function on some folder before serving locally.

**Comment by karlos1337** [comment]
How about running `invoke local -f` without transpiling if the file already exists?

The point is, using `serverless offline` should be started for invoking any task and already transpiles all ts files so have no sense doing again the same on each `invoke`, and if you want to rebuild on each change could use `serverless offline --watch` so...
